recipes/m2w64-jq/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "jq" %}
```

Comment:
Probably need to credit the original author. I think the msys2 channels are BSD-3-Clause right?